### PR TITLE
Handle rsa keys when signing hotspot txns.

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: src/features/txnDelegation/useSolTxns.ts [hunk 1]
```diff
@@ -139,6 +144,22 @@ const useSolTxns = (heliumAddress: string, solanaTransactions?: string) => {
     [],
   )
 
+  const assetToAddress = useCallback(
```
Comment (ChewingGlass) [inline src/features/txnDelegation/useSolTxns.ts:147]: There's a function in `helium-entity-manager-sdk` that does this. Should use that so we don't have to maintain this gross logic in multiple places

**Review by ChewingGlass — CHANGES_REQUESTED** [review]

File: src/features/txnDelegation/useSolTxns.ts (hunk 1 above)
Comment (ChewingGlass) [reply src/features/txnDelegation/useSolTxns.ts:147]: @matthewcarlreetz bump on this, looks like this feature is needed

Comment (matthewcarlreetz) [reply src/features/txnDelegation/useSolTxns.ts:147]: @ChewingGlass what function are you referring to?

Comment (matthewcarlreetz) [reply src/features/txnDelegation/useSolTxns.ts:147]: I'm already using `import { decodeEntityKey } from '@helium/helium-entity-manager-sdk'`

**Review by ChewingGlass — APPROVED** [review]

**Pull request merged**